main_app/views.py

```diff
@@ -74,20 +72,57 @@ class CategoryUpdate(UpdateView):
 class CategoryDelete(DeleteView):
     model = Category
     success_url = '/categories'
-    
+
+# item stuff    
 class ItemCreate(CreateView):
     model = Item
     fields = ['name', 'description', 'quantity_current','quantity_max']
     success_url = '/'
 
+    def form_valid(self, form):
```

Comment:
form_valid() function here to make sure that the details from the action get read in the history log